bin/mcb

```diff
@@ -36,6 +37,11 @@ $mcb = Cri::Command.define do
 
   flag :h, :help, 'show help for this command' do |_value, cmd|
     puts cmd.help
+    if $verbosity == :verbose
+      puts
+      puts Rainbow("SUBCOMMANDS").red.bright
+      show_all_commands(cmd, "")
+    end
```

Comment:
Just a guess, but this won't work if the command switches are done in the order: -h -v, will it? That's a bother, if true. :(

Reply:
You are correct sir, I have no idea what to do about it....

Reply:
Kicks can down road... :-)